errors/errors.md

```diff
@@ -0,0 +1,28 @@
+# How to handle errors
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged